scripts/diff-coverage.py

```
@@ -121,13 +121,17 @@ def parse_unified_diff(text: str) -> dict[str, list[int]]:
         m = _HUNK_RE.match(line)
         if m:
             if current is None:
-                raise ToolError(f"hunk 出现在未识别文件头之后: {line!r}")
+                # 删除文件的 hunk（+++ /dev/null 之后）：合法形态——无新行，消费即可
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)